pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -13,8 +13,11 @@
 , cupsSupport ? true
 , pulseSupport ? config.pulseaudio or stdenv.isLinux
 , commandLineArgs ? ""
+, VAAPISupport ? false, libva ? null
```

Comment:
I maintain we would want to enable this by default in versions >73, otherwise almost noone would be able to use it given the extremely long compile times.

Reply:
I have no strong opinion on that. Would create an alias (something like "chromium-vaapi") force the builders to cache a binary ?
Another solution would be to condition it on libva presence in `hardware.opengl.extraPackages`.

Reply:
Creating a separate alias would cause Hydra to build Chromium twice - probably not something we want, as the evaluations already take long enough.
It's not possible (and shouldn't be) to incorporate NixOS config values in Nix packages.

Reply:
> Creating a separate alias would cause Hydra to build Chromium twice - probably not something we want, as the evaluations already take long enough.

Maybe. Anyway, I guess this decision is the package maintainers to make. @bendlas ?

> AFAIK it's not possible (and shouldn't be) to incorporate NixOS config values in Nix packages.
Indeed, I looked at it from a nixos centric pov. You're right.

Reply:
I'm fine with enabling by default on master. I'd leave it disabled by default on 19.03 (for now).